css/modal.css

```diff
@@ -0,0 +1,51 @@
+.modal{
+    background-color: var(--background-alpha);
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 100vw;
+    overflow-y: scroll;
+}
+
+.modal.open{
+    display: flex;
+}
+
+.modal-dialog{
+    margin: 2rem auto; 
+    height: max-content;
+    background-color: var(--background-primary);
+    border-radius: 4px;
+    min-width: 300px;
```

Comment:
you can use 'rem' here.

Reply:
Thanks for the review. i'll change this.